app/js/components/filterComponents/actionsSelect/actionsSelect.directive.js

```diff
@@ -36,21 +37,31 @@ function actionsSelectCtrl($rootScope,
 
     $scope.select = function (option) {
         $scope.selected = option;
-        MultiButtonService.setState('inventoryWithActions', option.withActions);
-        MultiButtonService.setState('inventoryWithoutActions', option.withoutActions);
+        setSystemHealth(option);
         FilterService.doFilter();
         $rootScope.$broadcast(Events.filters.tag, getTag(), Events.filters.actionsSelect);
     };
 
+    function setSystemHealth(option) {
+        $location.search('withActions', option.withActions.toString());
+        $location.search('withoutActions', option.withoutActions.toString());
+        MultiButtonService.setState('inventoryWithActions', option.withActions);
+        MultiButtonService.setState('inventoryWithoutActions', option.withoutActions);
+    }
```

Comment:
Let's get rid of this method as well as all of the MultiButtonService references. I think the MultiButtonService is obsolete at this point. It was used for filters with multi-states (ie. filters that could hold two states at one time). Can you rewrite this filter to match the other filter components?